wurst/heroes/heroes/DemonLord/HowlOfTerror.wurst

```diff
@@ -2,25 +2,37 @@ package HowlOfTerror
 import Abilities
 import AbilityTooltipGenerator
 import BuffIds
-import ClosureEvents
+import DamageEvent
 import DemonLordIds
 import GameConstants
 import Icons
+import StandardTextTags
 import TooltipFactory
 
 RealLevelClosure DAMAGE_DECREASE = lvl -> 0.25 + 0.1 * (lvl-1)
 RealLevelClosure DURATION = lvl -> 10
-IntLevelClosure STR_BONUS = lvl -> (0.5 * (lvl - 1)).floor() + 1 // 1, 1, 2, 2, 3
+IntLevelClosure STR_BONUS = _ -> 1
 RealLevelClosure HP_RECOVERY = lvl -> 100.0 + 50 * (lvl-1)
 
 init
-  EventListener.add(EVENT_PLAYER_UNIT_DEATH) ->
-    let killer = EventData.getKillingUnit()
-    if killer.getTypeId() == DEMON_LORD_ID and EventData.getTriggerUnit().hasAbility(BuffIds.howlofTerror)
-      let level = killer.getAbilityLevel(HOWL_OF_TERROR_ID)
-      killer.addStr(STR_BONUS.run(level))
-      killer.addHP(HP_RECOVERY.run(level))
-      flashEffect(Abilities.deathCoilMissile, killer.getPos())
+  DamageEvent.addListener(5) ->
```

Comment:
Old code was not working - EVENT_UNIT_DEATH event fires after buffs are removed from unit

Reply:
![image](https://user-images.githubusercontent.com/29690395/138141955-af26cd3d-a972-446a-99f5-51b9368df50b.png)
but why is this part missing? I added aggro and fixed that bug and frotty fixed addHp to custom heal system